understand_sentiment/README.en.md

```diff
@@ -0,0 +1,491 @@
+
+| Movie Review       | Category  |
+| --------     | -----  |
+| Best movie of Xiaogang Feng in recent years!| Positive |
+| Pretty bad. Feels like a tv-series from a local TV-channel     | Negative |
+| Politically correct version of Taken ... and boring as Heck| Negative|
+|delightful, mesmerizing, and completely unexpected. The plot is nicely designed.|Positive|
+
+<p align="center">Table 1 Sentiment Analysis in Movie Reviews</p>
+
+In natural language processing, sentiment analysis can be categorized as a **Text Classification problem**, i.e., to categorize a piece of text to a specific class. It involves two related tasks: text representation and classification. Before deep learning becomes heated, the main-stream methods for the former include BOW (bag of words) and topic modeling, while the latter contain SVM(support vector machine), LR(logistic regression).
+
+For a piece of text, BOW model ignores its word order, grammar and syntax, and regard it as a set of words, so BOW does not capture all the information in the text. For example, “this movie is extremely bad“ and “boring, dull and empty work” describe very similar semantic with low similarity in sense of BOW. Also, “the movie is bad“ and “the movie is not bad“ have high similarity with BOW feature, but they express completely opposite semantics.
+
+
+In this chapter, we introduce our deep learning model which handles these issues in BOW. Our model embeds texts into a low-dimensional space and takes word order into consideration. It is an end-to-end framework, and has large performance improvement over traditional methods \[[1](#Reference)\].
+
+## Model Overview
+The model we used in this chapter is the CNN (Convolutional Neural Networks) and RNN (Recurrent Neural Networks) with some specific extension. 
+
+
+### Convolutional Neural Networks for Texts (CNN)
+Convolutional Neural Networks are always applied in data with grid-like topology, such as 2-d images and 1-d texts. CNN can combine extracted multiple local features to produce higher-level abstract semantics. Experimentally, CNN is very efficient for image and text modeling.
+
+CNN mainly contains convolution and pooling operation, with various extensions. We briefly describe CNN here with an example \[[1](#Refernce)\]. As shown in Figure 1：
+
+
+<p align="center">
+<img src="image/text_cnn.png" width = "80%" align="center"/><br/>
+Figure 1. CNN for text modeling
+</p>
```

Comment:
If there is text in this figure, could you please add here a list of need to be translated.